src/EditorFeatures/Core/Shared/Options/FeatureOnOffOptions.cs

```diff
@@ -84,6 +84,12 @@ internal static class FeatureOnOffOptions
                 defaultValue: true,
                 new RoamingProfileStorageLocation("TextEditor.%LANGUAGE%.Specific.ShowInheritanceMargin"));
 
+        public static readonly PerLanguageOption2<bool> InheritanceMarginCombinedWithIndicatorMargin =
```

Comment:
i wouldn't really expect this to be a per-language option.  is there a reason for that?  or should we just have this be a normal option that applies to all languages.  as a user, i dont' tend to think of changing UIs like this on a per-language basis... but i could be convinced otherwise.